packages/imu_driver/include/hardware_test_imu/hardware_test_imu.py

```diff
@@ -0,0 +1,66 @@
+import rospy
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+
+
+class HardwareTestIMU(HardwareTest):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def test_id(self) -> str:
+        return f"IMU"
+
+    def test_description_preparation(self) -> str:
+        return self.html_util_ul(
+            [
+                "Put your Duckiebot in its normal orientation, on a surface (e.g. desk).",
+                "Make sure you can reach and grab the robot when the test starts.",
+            ]
+        )
+
+    def test_description_running(self) -> str:
+        return self.html_util_ul(
+            [
+                "Hold your Duckiebot up a little bit off the surface.",
+                "Click on the <strong>Run the test</strong> button below to start the test.",
+            ]
+        )
+
+    def test_description_expectation(self) -> str:
+        return self.html_util_ul(
+            [
+                "Once the test starts, a modal would pop up with a game.",
```

Comment:
Once the test starts, a modal will pop up with a game.  You will see a flat plane reflecting your Duckiebot's orientation - your objective is to move the ball while keeping it on the plane.  

To play, control the plane by picking up and tilting your Duckiebot.  The test is successful if the plane moves naturally with the rotation of your Duckiebot.